jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRolesViewImpl.java

```diff
@@ -67,14 +74,27 @@
 
     List allElementsList = new ArrayList();
 
-    private Command userAddCommand;
+    private Command filterCommand;
 
     @Inject
     private ManagedInstance<CaseRoleItemView> provider;
 
+    @Inject
+    private TranslationService translationService;
+
     @Override
     public void init(final CaseRolesPresenter presenter) {
+    }
+
+    @PostConstruct
+    public void init() {
+        stream(CaseRolesAssignmentFilterBy.values())
+                .collect(toMap(s -> s.getLabel(),
+                        s -> translationService.format(s.getLabel())))
+                .entrySet()
+                .forEach(s -> filterSelect.addOption(s.getValue(), s.getKey()));
```

Comment:
The intermediate map does not seem to bring value. Why not do the `forEach()` directly on the first stream, saving the cycles and memory it costs to create the map?